packages/core/docs/guide/checkout.md

```diff
@@ -0,0 +1,145 @@
+# Checkout
+Checkout is a process of providing information about shipping and billing details, and picking certain shipping and payment method. Then with all the data, we are able to place an order and pay for it.
+
+## Shipping
```

Comment:
as @NataliaTepluhina was suggesting let's have more descriptive subtitles (also for SEO)